docs/SECURITY.md

```diff
@@ -12,7 +12,7 @@ For a list of recent security commits, check [our GitHub commits prefixed with S
 
 Discourse uses the PBKDF2 algorithm to encrypt salted passwords. This algorithm is blessed by NIST. Security experts on the web [tend to agree that PBKDF2 is a secure choice](https://security.stackexchange.com/questions/4781/do-any-security-experts-recommend-bcrypt-for-password-storage).
 
-**options you can customise in your production.rb file**
+**options you can customize in your production.rb file**
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617140210

I'm generally not enforcing en-US v en-GB in this PR, however in cases where one flavor or the other is strongly dominant, I have sometimes made changes